src/modules/DialogOverlay.vue

```diff
@@ -63,7 +65,7 @@ export default defineComponent({
       required: true,
     },
     accordionData: {
-      type: String,
+      type: Object as () => TermProps,
```

Comment:
Suggestion (nit): If this syntax has the purpose of casting the `Object` as `TermProps`, using the built-in `PropType` might be a more readable alternative. Feel free to disregard if this is a style choice.

Example:
`Object as PropType<TermProps>`

More info at: [Annotating Props](https://v3.vuejs.org/guide/typescript-support.html#annotating-props)